packages/davinci-client/src/lib/davinci.api.ts

```diff
@@ -250,7 +252,7 @@ export const davinciApi = createApi({
             clientId: state?.config?.clientId,
             login: 'redirect', // TODO: improve this in SDK to be more semantic
             redirectUri: state?.config?.redirectUri,
-            responseType: state?.config?.responseType,
+            responseType: state?.config?.responseType as 'code',
```

Comment:
This is casted because in our config slice, the action types responseType as a string where i have it as a union of two items. can be updated

Reply:
I'm having trouble tracing where we know what the response type is here. In config.slice it defaults to `code` if there is no response type sent in the action payload. It can also be code if it's explicitly set. How can we be certain the response type in `start` will be `code`?